test/test/boot.js

```diff
@@ -485,9 +485,6 @@ async function setupTestEnvironment() {
   disableScrollbars();
   workAroundLegacyEdgePromiseIssues();
 
-  await loadImaScript();
```

Comment:
Please also delete the definition of these.

Reply:
Done!